ganttproject/src/main/java/net/sourceforge/ganttproject/calendar/CalendarEditorPanel.java

```diff
@@ -303,12 +304,30 @@ public Component getTableCellRendererComponent(
     AbstractTableAndActionsComponent<CalendarEvent> tableAndActions = new AbstractTableAndActionsComponent<>(table) {
       @Override
       protected void onAddEvent() {
-        int lastRow = tableModel.getRowCount() - 1;
-        Rectangle cellRect = table.getCellRect(lastRow, 0, true);
-        table.scrollRectToVisible(cellRect);
-        table.getSelectionModel().setSelectionInterval(lastRow, lastRow);
-        table.editCellAt(lastRow, 0);
-        table.getEditorComponent().requestFocus();
+        LocalizedString title = InternationalizationKt.getRootLocalizer().create("add");
+        DialogKt.dialog(title, controller -> {
+
+          MultiDatePicker multiDatePicker = new MultiDatePicker();
+          multiDatePicker.setValue(LocalDate.now());
+
+          controller.setContent(multiDatePicker.getPopupContent());
+
+          controller.setupButton(ButtonType.APPLY, button -> {
+            button.setText(InternationalizationKt.getRootLocalizer().formatText("add"));
+            button.addEventFilter(javafx.event.ActionEvent.ACTION, event -> {
+              List < LocalDate > selectedDates = multiDatePicker.getSelectedDates();
```

Comment:
too many spaces around the type argument :)
also, you may like `var` syntax: `val selectedDates = multiDatePicker.getSelectedDates();`

Reply:
Removed